dev-libs/libjwt/libjwt-1.15.3.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="JWT C Library"
+HOMEPAGE="https://github.com/benmcollins/libjwt"
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/benmcollins/libjwt"
+else
+	SRC_URI="https://github.com/benmcollins/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="MPL-2.0"
+SLOT="0"
+IUSE="gnutls ssl"
+
+REQUIRED_USE="
+	|| ( ssl gnutls )
+"
+RDEPEND="
+	ssl? (
+		>=dev-libs/openssl-0.9.8:0=
```

Comment:
All versions of openssl in the tree is newer than this, so simply dev-libs/openssl:= I reckon?

Reply:
that version was drawn from the upstream configure.ac